### ISSUE TITLE
Add a field to OBO site with metadata field ‘preferred mechanism for attribution’

### ISSUE THREAD

**Issue by nicolevasilevsky** [OBOFoundry/OBOFoundry.github.io#1558]
@mellybelly mentioned that it would be nice to have a standardized way to represent the preferred mechanism for attribution for each ontology on the OBO website.

Suggestion: Add a field to OBO site with metadata field ‘preferred mechanism for attribution’

Is this something we could do?

**Comment by cthoyt** [comment]
Some ontologies already have a citation field. Is this a generalization of that?

**Comment by nicolevasilevsky** [comment]
Yeah, I think so. I guess a specific citation could be the preferred mechanism for attribution, or maybe it could be something else (like if an ontology didn't have any citation to reference, they could specify they want you to cite their GitHub repo or something?)

**Comment by matentzn** [comment]
I agree that is a good thing to have. Currently, we have:

```
"publications": {
			"level": "info",
			"description": "'publications' is an informative field. It specifies where publications about the ontology can be found.",
````

In the schema (no field `citation`). I would like a `cite` field, like:

```
"cite": {
			"level": "info",
			"description": "Preferred citation/attribution mechanism, for example a dataset DOI, a publication (PMID) or a CITATION.cff file in a GitHub repository.",
```


I am wondering whether we should require this to be a URL, to do some minimal QC on the field. Else it will be hard to restrict..

**Comment by jsstevenson** [comment]
(Not to bump a zombie issue, but...) alternatively -- what about being able to flag an individual `publication` with `preferred_attribution: true` + some additional styling on the ontology detail page to indicate that preferential status?

**Comment by matentzn** [comment]
@jsstevenson its a good thought, but what about the case where someone does not have a publication and wants to say: please site my GitHub repo, or, please cite my Zenodo doi?

**Comment by jsstevenson** [comment]
yeah, that's definitely true @matentzn. 

I clicked around the various citation-related sections of the site today (which is how I found this issue) and noticed a few small possible changes -- wrote them in a draft here: https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2610. I could also try to mock up something for this issue if that would be helpful.

**Comment by matentzn** [comment]
Awesome! Thanks a ton! Your help is much appreciated!
